packages/consent-manager/util/cookies.test.js

```diff
@@ -34,19 +34,17 @@ it('should load preferences', () => {
 })
 
 it('should save preferences', () => {
+  global.window.location = { hostname: 'foo.bar.baz.net' }
```

Comment:
Is this `consent-manager` change intended?

Reply:
Yeah, the test was failing after I had to update a dependency to fix another error so made this to fix!